pkg/plugin/plugin.go

```diff
@@ -377,6 +377,21 @@ func getOvsPortForContIface(ovsDriver *ovsdb.OvsBridgeDriver, contIface string,
 	return ovsDriver.GetOvsPortForContIface(contIface, contNetnsPath)
 }
 
+// cleanPorts removes all ports whose interfaces have an error.
+func cleanPorts(ovsDriver *ovsdb.OvsBridgeDriver) {
+	ifaces, err := ovsDriver.FindInterfacesWithError()
+	if err != nil {
+		log.Printf("Error: %v\n", err)
```

Comment:
This should return an error. That would be later handled by the caller (logged).

Reply:
Changed :slightly_smiling_face: